Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ButtonGroup.cpp

```diff
@@ -28,6 +29,21 @@ namespace AzToolsFramework::ViewportUi::Internal
         return m_viewportUiId;
     }
 
+    void ButtonGroup::SetDisabledButton(ButtonId buttonId, bool isDisabled)
+    {
+        if (auto buttonEntry = m_buttons.find(buttonId); buttonEntry != m_buttons.end())
+        {
+            switch (buttonEntry->second->m_state)
+            {
+            case Button::State::Selected:
+                ClearHighlightedButton();
```

Comment:
update: Do we want a `break` here? If not can you please add `[[fallthrough]]` to make it clear that this is the intended behavior